docs/templates.md

```diff
@@ -318,6 +318,11 @@ defined.
 * `.substr(start: Integer, end: Integer) -> String`: Extract substring. The
   `start`/`end` indices should be specified in UTF-8 bytes. Negative values
   count from the end of the string.
+* `.escape_json() -> String`: Serializes the string in json format. This
```

Comment:
same here: capitalize "JSON" (that seems to be the official captilization) and hyphenate "machine-readable"